### PR TITLE
Adjust AUR package to project rename, add completion files

### PR DESCRIPTION
**Pull request by jneidel**
New AUR package is ready: https://aur.archlinux.org/packages/obsidian-cli-bin

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -40,18 +40,23 @@ scoops:
...
     maintainers:
       - 'Kartikay Jainwal <kartikayjainwal at gmail dot com>'
       - 'Jonathan Neidel <aur at jneidel dot com>'
     license: 'MIT'
     private_key: '{{ .Env.AUR_KEY }}'
-    git_url: 'ssh://aur@aur.archlinux.org/obs-bin.git'
+    git_url: 'ssh://aur@aur.archlinux.org/obsidian-cli-bin.git'
     package: |-
-      install -Dm755 "./obs" "${pkgdir}/usr/bin/obs"
-      install -Dm644 "./LICENSE" "${pkgdir}/usr/share/licenses/${pkgname}/LICENSE"
```
Comment (danielwerg) [inline .goreleaser.yml:54]: License removed on purpose?

Comment (jneidel) [reply .goreleaser.yml:54]: Yeah, comment on the package suggested it 🙂

Comment (jneidel) [reply .goreleaser.yml:54]: I tested the completions locally, works fine

**Review by danielwerg — APPROVED** [review]

**Review by Yakitrak — APPROVED** [review]
LGTM - thank you

**Pull request merged**